cmd/root.go

```diff
@@ -49,7 +50,7 @@ HPC deployments on the Google Cloud Platform.`,
 				log.Fatalf("cmd.Help function failed: %s", err)
 			}
 		},
-		Version:     "v1.13.0",
+		Version:     "v1.14.0",
```

Comment:
nit: I kind of like putting the release notes in the PR description like #916, but not required.

Reply:
Makes sense, updated commit message